chain/chain/src/flat_storage_resharder.rs

```diff
@@ -501,7 +501,7 @@ pub enum FlatStorageReshardingTaskStatus {
 
 /// Helps control the flat storage resharder operation. More specifically,
 /// it has a way to know when the background task is done or to interrupt it.
-#[derive(Clone)]
+#[derive(Clone, Debug)]
```

Comment:
I'm surprised that sender and receiver implement Debug but alright.

Reply:
Caught me by surprise as well